dts/bindings/memory-controllers/nuvoton,npcx-bbram.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 20201 Google Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX Battery Backed RAM node
+
+compatible: "nuvoton,npcx-bbram"
+
+include: base.yaml
+
+properties:
+    "#address-cells":
+        required: false
+        const: 1
+    "#size-cells":
+        required: false
+        const: 1
```

Comment:
Do you plan on adding sub-nodes to this? Otherwise I don't think you need to specify these two.